src/api-client/server/index.ts

```diff
@@ -53,9 +68,27 @@ export default {
       })
       .catch(catchError);
   },
-  async getUser(token: string, user: string): Promise<InternalActor> {
+  async updateProfile(
+    token: string,
+    user: string,
+    updateProfile: UpdateProfile
+  ): Promise<InternalActor> {
+    return await axios
+      .patch(`${urlPrefix}/user/${user}`, updateProfile, {
+        headers: {
+          Accept: "application/json",
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${token}`,
```

Comment:
Why is one with and the other without double quotes?

Reply:
The divis cannot appear in a property without quotes, since only alphanumeric characters and underline and dollar signs are allowed without quotes.
The Lint complains about the quotes for alphanumeric property. I would have to adjust the lint rule in the configuration file. But for other properties that could no longer indicate that no quotes are allowed. I could also switch off the lint rule with a comment above the line